setup.py

```diff
@@ -31,7 +31,7 @@ def get_version(short=False):
       url='https://github.com/williamjameshandley/anesthetic',
       packages=find_packages(),
       scripts=['scripts/anesthetic'],
-      install_requires=['numpy', 'scipy>=1.2.0', 'pandas', 'matplotlib>=3.1.2'],
+      install_requires=open('requirements.txt').read().splitlines(),
```

Comment:
Do we need a similar `install_requires` for `requirements_extras.txt`?

Reply:
It depends whether or not we want this as a dict (as it is at the moment), so that you can install extras individually as necessary, or whether we want it more automated and just have it as a (filtered) list. I would vote for keeping it as-is at the moment.